cmd/cloud-controller-manager/main.go

```diff
@@ -68,7 +68,11 @@ func main() {
 	nodeIpamController.nodeIPAMControllerOptions.NodeIPAMControllerConfiguration = &nodeIpamController.nodeIPAMControllerConfiguration
 	fss := cliflag.NamedFlagSets{}
 	nodeIpamController.nodeIPAMControllerOptions.AddFlags(fss.FlagSet("nodeipam controller"))
-	controllerInitializers["nodeipam"] = nodeIpamController.startNodeIpamControllerWrapper
```

Comment:
I think you accidentally swapped node-ipam with cloud-node here, is that intentional?

Reply:
Nope, thanks.